integrationtests/python_src/px4_it/dronekit/MissionCheck.py

```diff
@@ -227,11 +162,8 @@ def listener(self, name, home_position):
 missionlist = upload_mission(import_mission_filename)
 time.sleep(2)
 
-# set mission mode the hard way
-vehicle._master.mav.command_long_send(vehicle._master.target_system, vehicle._master.target_component,
-                                           mavutil.mavlink.MAV_CMD_DO_SET_MODE, 0,
-                                           MAV_MODE_AUTO,
-                                           0, 0, 0, 0, 0, 0)
+# set mission mode
+vehicle.mode = VehicleMode("MISSION")
```

Comment:
This won't work on the current pyMAVLink version - can we keep the "hard way" until that's fully merged?